src/components/Messages/ChatMenu.vue

```diff
@@ -0,0 +1,221 @@
+        <v-autocomplete
+          dense
+          class="mx-4 text-body-2"
+          flat
+          hide-no-data
+          hide-details
+          label="Search for people and messages"
+          outlined
+          color="primary"
+          hide-selected
+          single
+          line
+          return-object
+          single-line
+          append-icon="mdi-magnify"
+          data-cy="search"
+        ></v-autocomplete>
+      </v-col>
+      <v-col cols="12" align-self="center" class="px-8 pb-6">
+        <v-btn small color="primary" width="100%" data-cy="new-message">
+          <v-icon left>
+            mdi-pencil
+          </v-icon>
+          New Message
+        </v-btn>
+      </v-col>
+    </v-row>
+    <v-divider />
+    <v-responsive class="overflow-y-auto fill-height" height="65vh">
+      <v-list subheader>
+        <v-list-item
+          v-for="user in chats"
+          v-bind:key="user.name"
+          :to="/messages/ + user.id"
+          data-cy="chat"
+          class="px-0"
+          @click="
+            conversationSelected(
+              user.name,
+              user.username,
+              user.img,
+              user.messages
+            )
+          "
+        >
+          <v-list-item :key="`user${index}`" :value="user.id">
+            <v-avatar class="profile" size="40">
+              <v-img :src="user.img"></v-img>
+            </v-avatar>
+            <v-list-item-content class="text-left pl-5">
+              <v-col class="px-0 py-0" cols="6">
+                <v-list-item-title
+                  v-text="user.name"
+                  class="text-wrap text-body-2"
+                />
+              </v-col>
+              <v-col class="px-0 py-0" cols="6">
+                <v-list-item-subtitle
+                  v-text="user.timeLastMessage"
+                  class="text-wrap text-right text-caption"
+                />
+              </v-col>
+              <v-col class="px-0 py-0">
+                <v-list-item-subtitle
+                  v-text="user.lastMessage"
+                  class="pt-2 text-caption text-truncate"
+                  style="max-width: 19vw;"
+                />
+              </v-col>
+            </v-list-item-content>
+          </v-list-item>
+        </v-list-item>
+      </v-list>
+    </v-responsive>
+  </v-container>
+</template>
+
+<script>
+  export default {
+    methods: {
+      conversationSelected(name, username, img, messages) {
+        this.$emit("clicked", name, username, img, messages);
+      },
+    },
+    computed: {
+      chats() {
+        return [
+          {
+            id: 1,
+            name: "John Doe",
+            username: "@johndoe",
+            img: "https://randomuser.me/api/portraits/men/52.jpg",
+            lastMessage: "What are you up to today?",
+            timeLastMessage: "5m",
+            messages: [
+              {
+                content: "Hey! How have you been",
+                me: true,
+              },
```

Comment:
I don't think it is necessary in this PR but I think it would be beneficial to have a timestamp on each message and the timeLastMessage should be calculated from the difference of the timestamp from the last message and a current timestamp. I think that's how it is going to be in the backend but also if you did it now it would be make the integration quicker when the backend part is done

Reply:
Yeah I can add that on my next PR. Would it be just in the data structure, and not visible on the page?